main/gpgme/APKBUILD

```diff
@@ -40,8 +42,27 @@ check() {
 gpgmepp() {
 	pkgdesc="C++ bindings for GPGME"
 	mkdir -p "$subpkgdir"/usr/lib
-	mv "$pkgdir"/usr/lib/libgpgmepp.so.* "$subpkgdir"/usr/lib/
+	cp "$pkgdir"/usr/lib/libgpgmepp.so.* "$subpkgdir"/usr/lib/
```

Comment:
why is this changed to `cp`? it means that `libgpgmepp.so.*` will be provided by both main package and gpgmepp, which I don't think we want.